sdv/single_table/copulas.py

```diff
@@ -323,12 +324,14 @@ def _rebuild_correlation_matrix(cls, triangular_correlation):
 
         return cls._get_nearest_correlation_matrix(correlation).tolist()
 
-    def _rebuild_gaussian_copula(self, model_parameters):
+    def _rebuild_gaussian_copula(self, model_parameters, default_params={}):
```

Comment:
Can we change `default_pramas={}` to `None` and create the dict inside if required.
Here is why:
<img width="502" alt="image" src="https://github.com/sdv-dev/SDV/assets/41479552/cfd9f1cf-51c0-4996-8c6b-0ab500051f62">